v19.2/learn-cockroachdb-sql-movr.md

```diff
@@ -0,0 +1,381 @@
+---
+title: Learn CockroachDB SQL
+summary: Learn some of the most essential CockroachDB SQL statements on a local cluster.
+toc: true
+build_for: [standard, managed]
+---
+
+This tutorial walks you through some of the most essential CockroachDB SQL statements, using the `movr` dataset. For a complete list of supported SQL statements and related details, see [SQL Statements](sql-statements.html). For more information about the `movr` dataset and the MovR example application, see [MovR: A Global Ride-sharing App](movr.html).
+
+{{site.data.alerts.callout_info}}
+CockroachDB aims to provide standard SQL with extensions, but some standard SQL functionality is not yet available. See our [SQL Feature Support](sql-feature-support.html) page for more details.
```

Comment:
I think this note was already here, but I've filed https://github.com/cockroachdb/docs/issues/5125 to try to remove it

Reply:
OK. I'll go ahead and remove it from here.